examples/custom-field/stars-field/index.ts

```diff
@@ -0,0 +1,120 @@
+import {
+  BaseGeneratedListTypes,
+  FieldDefaultValue,
+  fieldType,
+  FieldTypeFunc,
+  CommonFieldConfig,
+  legacyFilters,
+  orderDirectionEnum,
+  schema,
+} from '@keystone-next/types';
+
```

Comment:
Can we add a comment here saying that this is all based mostly on the `integer` field type, and include a link so that the reader can go an compare what's happening here to what's happening there?